src/components/MovieList.js

```diff
@@ -1,22 +1,25 @@
-import React from 'react';
+import React, { Component } from 'react';
 import Movie from './Movie';
 
-const MovieList = ({ movies }) =>
-  (
-    <div>
-      {
-        movies.map(({poster, id, title, genre}, i) => (
-            <Movie
-              key={i}
-              id={id}
-              title={title}
-              genre={genre}
-              poster={poster}
-              />
+class MovieList extends Component{
+  render(){
+    const {movies} = this.props;
+    return (
+      <div>
+        {
+          movies.map(({Poster, imdbID, Title, Year}, i) => (
```

Comment:
dont use capitalized letters, i'd rather have `movie.Title` if it has to do w/ the data structure coming from the API